src/report/get-report.ts

```diff
@@ -128,24 +137,29 @@ function getBadge(passed: number, failed: number, skipped: number): string {
     color = 'yellow'
   }
   const hint = failed > 0 ? 'Tests failed' : 'Tests passed successfully'
-  const uri = encodeURIComponent(`tests-${message}-${color}`)
+  const uri = encodeURIComponent(`${options.badgeTitle}-${message}-${color}`)
   return `![${hint}](https://img.shields.io/badge/${uri})`
 }
 
 function getTestRunsReport(testRuns: TestRunResult[], options: ReportOptions): string[] {
   const sections: string[] = []
+  const totalFailed = testRuns.reduce((sum, tr) => sum + tr.failed, 0)
+  if (totalFailed === 0) {
+    sections.push(`<details><summary>Expand for details</summary>`)
+    sections.push(` `)
+  }
 
-  if (testRuns.length > 1 || options.onlySummary) {
-    const tableData = testRuns.map((tr, runIndex) => {
-      const time = formatTime(tr.time)
-      const name = tr.path
-      const addr = options.baseUrl + makeRunSlug(runIndex).link
-      const nameLink = link(name, addr)
-      const passed = tr.passed > 0 ? `${tr.passed}${Icon.success}` : ''
-      const failed = tr.failed > 0 ? `${tr.failed}${Icon.fail}` : ''
-      const skipped = tr.skipped > 0 ? `${tr.skipped}${Icon.skip}` : ''
-      return [nameLink, passed, failed, skipped, time]
-    })
+  if (testRuns.length > 0 || options.onlySummary) {
+    const tableData = testRuns
+      .filter(tr => tr.passed > 0 || tr.failed > 0 || tr.skipped > 0)
+      .map(tr => {
+        const time = formatTime(tr.time)
+        const name = tr.path
+        const passed = tr.passed > 0 ? `${tr.passed}${Icon.success}` : ''
+        const failed = tr.failed > 0 ? `${tr.failed}${Icon.fail}` : ''
+        const skipped = tr.skipped > 0 ? `${tr.skipped}${Icon.skip}` : ''
```

Comment:
There should be space between the number and the icon. Otherwise GitHub markdown will not render it correctly.

See this file: https://github.com/ritchxu/test-reporter/blob/ritchxu/support-actions-summary/__tests__/__outputs__/swift-xunit.md

| markdown | rendering |
|------------------------|----------------------|
| `2:white_check_mark:`  | 2:white_check_mark:  |
| `2 :white_check_mark:` | 2 :white_check_mark: |